### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by akashchandwani**

### CODE REVIEW

File: docs/landing-page.md [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+Welcome to our product page of Open Banking Accelerator. Open Banking Accelerator provides a set of tools and resources which you can use to start your open banking journey and achieve the open banking compliant status fast. Our Aim is to accelerate your Open banking journey with a blinding speed.
 
-### Our APIs are perfect for
+## How do we do it?
+
+We provide a set of tools out of the box which you can plug and play to get started quickly. The tools are as follows -
+
+1. Backstage Scaffolder - with the power of backstage scaffolder, you can create your initial code respositories in your remote organization repositories very quickly with all the starter code already written. You just have to integrate with your backend services. You can choose to scaffold the following templates from the create menu on the left hand side of the Backstage page -
+
+   1. OBA-Spec_service - This provides a adapter service which accepts open banking APIs and returns the responses in the desired format. The services should call the bank's internal APIs in Core banking systems to get the appropriate response. The conversion will be different for the each bank.
```
Comment (Paridhi-Mohindra) [inline docs/landing-page.md]: This should be point number 2.

File: docs/landing-page.md [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
+   2. OBA-Kong_gateway - Kong is a popular gateway service. All the APIs reaching OBA-Spec_service pass through it.
```
Comment (Paridhi-Mohindra) [inline docs/landing-page.md]: This should be point number 3.

File: docs/landing-page.md (hunk 1 above)
Comment (akashchandwani) [reply docs/landing-page.md]: This is sub numbering

File: docs/landing-page.md (hunk 2 above)
Comment (akashchandwani) [reply docs/landing-page.md]: This is sub numbering

**Pull request merged**